bedrock/firefox/views.py

```diff
@@ -679,6 +678,8 @@ def get_template_names(self):
             # return a list to conform with original intention
             return ['firefox/whatsnew/index-lite.html']
 
+        template = super().get_template_names()
```

Comment:
NIT: better to just `return super().get_template_names()`. No need for the extra variable.

Reply:
Unless you'd like to have a single return statement, which is nice stylistically. e.g.:

```python
        if locale.startswith('en-') and channel not in ['nightly', 'alpha', 'beta']:
            # return a list to conform with original intention
            templates = ['firefox/whatsnew/index-lite.html']
        else:
            templates = super().get_template_names()

        return templates